Exiled.API/Features/Intercom.cs

```diff
@@ -55,5 +64,21 @@ public static float SpeechRemainingTime
         /// <param name="start">Sets a value indicating whether or not the sound is the intercom's start speaking sound.</param>
         /// <param name="transmitterId">Sets the transmitterId.</param>
         public static void PlaySound(bool start, int transmitterId = 0) => global::Intercom.host.RpcPlaySound(start, transmitterId);
+
+        /// <summary>
+        /// Reset the intercom's cooldown.
+        /// </summary>
+        public static void Reset() => RemainingCooldown = -1f;
+
+        /// <summary>
+        /// Times out the intercom.
+        /// </summary>
+        public static void Timeout()
+        {
+            if(InUse)
```

Comment:
Leave a space after the `if` statement like: `if (InUse)`[...]

Reply:
done